index.js

```diff
@@ -113,6 +113,17 @@ server.route([
     }
   },
   {
+    path: '/users/{userId}/addProperty',
```

Comment:
We will want to use a better endpoint than this.

It should be a POST request to the `/properties` resource endpoint, as you are adding a property to the properties collection.

Can you have a little read up on RESTful design of web services so you understand why this is the case.